iron-doc-behavior.html

```diff
@@ -179,6 +182,10 @@
         return element.name || element.tagname;
       },
 
+      _getPolymerBehaviors: function(descriptor) {
```

Comment:
I think I'll keep it consistent with the rest of the repo for now. I'm planning a big sweep through to fixup style later.